VENTokenField/VENToken.h

```diff
@@ -29,6 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (assign, nonatomic) BOOL highlighted;
 @property (copy, nonatomic, nullable) void (^didTapTokenBlock) (void);
 @property (strong, nonatomic) UIColor *colorScheme;
+@property (strong, nonatomic) UIColor *backgroundColorWhenHighlighted;
```

Comment:
change name to `highlightedBackgroundColor`

did you mean that or `highlightedTextColor`?